kafka-lib_0_10/src/main/scala/Consumer.scala

```diff
@@ -6,6 +6,16 @@ import scala.util.Try
 trait Consumer[T] {
 
   def kafkaConsumer: KafkaConsumer[String, String]
+  
+  def processBatch(
```

Comment:
Happy to leave it for now though as long as we reflect this change in the version number.